tests/test_connectbox_static.py

```diff
@@ -70,35 +70,6 @@ def getAdminAuth():
     return requests.auth.HTTPBasicAuth(ADMIN_USER, ADMIN_PASSWORD)
 
 
-class ConnectBoxBasicTestCase(unittest.TestCase):
-
-    def testContentResponseType(self):
```

Comment:
why is this being removed? It's exercising the webserver rather than the interface code, so I'd expect it to stay.

Reply:
I removed these because /admin/ is no longer doing basic auth and the /admin/ path is handled by the react app now instead of the web server. I should probably add a test for /admin/api.py to validate basic auth, but the rest of these no longer apply.